gulp/test-browser.js

```diff
@@ -10,6 +10,23 @@ import commonjs from 'rollup-plugin-commonjs';
 import json from 'rollup-plugin-json';
 import opn from 'opn';
 
+const sauceConf = {
+  name: 'Marionette.js',
+  username: process.env.SAUCE_USERNAME,
+  key: process.env.SAUCE_ACCESS_KEY,
+  tests: '/test/runner.html',
```

Comment:
Yeah, looks like I missed a few instances in my find-and-replace. I've fixed it and released easy-sauce 0.4.1 with the changes.